bindings/nodejs/lib/types/block/public-key.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+// Temp solution for not double parsing JSON
```

Comment:
Temp? Should there be a TODO?

Reply:
huh yea this makes no sense, should be gone